internal/hcsoci/resources_wcow.go

```diff
@@ -19,11 +19,12 @@ import (
 	"github.com/Microsoft/hcsshim/internal/uvm"
 	"github.com/Microsoft/hcsshim/internal/wclayer"
 	specs "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/pkg/errors"
 )
 
-func allocateWindowsResources(ctx context.Context, coi *createOptionsInternal, r *resources.Resources) error {
+func allocateWindowsResources(ctx context.Context, coi *createOptionsInternal, r *resources.Resources, sandbox bool) error {
```

Comment:
same here with `isSandbox`

Reply:
Just gonna c+p the answer to the other. I didn't want to reparse the container type as we already do that earlier in CreateContainer. I'm fine to meet in the middle with isSandbox